src/Shell/Task/QueueLongExampleTask.php

```diff
@@ -75,7 +75,7 @@ public function add() {
 	 * @param array $data The array passed to QueuedTask->createJob()
 	 * @param int $id The id of the QueuedTask
 	 * @return bool Success
-	 * @throws RuntimeException when seconds are 0;
+	 * @throws Queue\Shell\Task\RuntimeException when seconds are 0;
```

Comment:
unlikely, only because you didnt put it into the namespace
but there is no such file anyway, so using \ prefix is sufficient.